### PR TITLE
(SIMP-6366) Configure Bolt module ssh

### PR DESCRIPTION
**Pull request by m-morrone**
SIMP-6366 #comment Create simplib__sshd_config fact for simp_bolt

### CODE REVIEW

File: spec/unit/facter/simplib__sshd_config_spec.rb [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+# Even inline comments!
+    # And indented comments
+      EOM
+    }
+    it 'should return hash of values from /etc/ssh/sshd_config' do
+      File.expects(:exist?).with('/etc/ssh/sshd_config').returns(true)
+      File.expects(:readable?).with('/etc/ssh/sshd_config').returns(true)
+      File.expects(:read).with('/etc/ssh/sshd_config').returns(sshd_config_content)
+
+      # This resets the stubbing code in Mocha to ensure that the code does not
```
Comment (sharkbruhaha) [inline spec/unit/facter/simplib__sshd_config_spec.rb]: should we be bothering to make a work around for Mocha, when it is deprecated? (spits out message about it when we run the tests)

Comment (m-morrone) [reply spec/unit/facter/simplib__sshd_config_spec.rb]: Mocha isn't deprecated; the warning is just to draw attention to the user that puppetlabs_spec_helper defaults to mocha.

Comment (sharkbruhaha) [reply spec/unit/facter/simplib__sshd_config_spec.rb]: ahhh

**Comment by trevor-vaughan** [comment]
:hourglass_flowing_sand: https://gitlab.com/simp/pupmod-simp-simp/pipelines/57618435

**Comment by m-morrone** [comment]
Changes make sense and permit simpler expansion of returned settings.

**Pull request merged**